app/Controllers/AddDonationController.php

```diff
@@ -242,4 +238,22 @@ private function resetSessionState(): void {
 			false
 		);
 	}
+
+	/**
+	 * @param ConstraintViolation[] $constraintViolations
+	 */
+	private function logValidationErrors( $constraintViolations ): void {
+		$formattedConstraintViolations = [];
+		foreach ($constraintViolations as $constraintViolation) {
```

Comment:
spacing is wrong. I think the CS stuff can be updated to catch this. IIRC, this was disabled since when we started it did not work with PHP 7.0

Reply:
Thanks, I will look into enabling this for our CS.

Reply:
The issue was actually that the directory is not included in the CS config.